src/utils/middlewares/azure_api_auth.ts

```diff
@@ -221,3 +222,60 @@ export const AzureApiAuthMiddleware = (
       )
     );
   });
+
+type AzureAllowBodyPayloadMiddlewareMiddlewareErrorResponses =
```

Comment:
There is a repetition in this type "...MiddlewareMiddleware...", is it desired?

Reply:
ooops, thanks!

Reply:
fixed in https://github.com/pagopa/io-functions-commons/pull/167/commits/498e40d9165fefd4dcfdac2cd0f3218a6909023c